### DISCUSSION TITLE
Letting the Scribble library be affected by the orbinaut shader for fading?

### DISCUSSION THREAD

**Discussion by ManeaterMildred** [TrianglyRU/OrbinautFramework#49] [Q&A] [answered]
Hey, rather small question this time around. It's not too important, but I do prefer using Scribble over draw_text.

For context, Scribble is a library for text drawing that incorporate a lot of features like typewriter, colors, adding image in text, etc etc. [Here is the link.](https://www.jujuadams.com/Scribble/#/latest/)

It's been working just fine in the framework here, except with just one issue: It's not being affected by the fadeout.

![image](https://github.com/user-attachments/assets/ab11443f-1c36-4bae-b3c9-1c546661049a)
![image](https://github.com/user-attachments/assets/beccfbf9-6faa-4384-a9b5-410fadd4a7b6)
Here, the HP number is being drawn by Scribble, and it doesn't want to, well, get faded. The draw code is just a part of obj_gui_hud, which has everything else be affected as it should.

So, yeah, curious if there's a way to add Scribble text to the fade function? If not, I can do without, but it would be appreciated: I prefer using Scribble for writing text a lot, haha.

**Comment by TrianglyRU** [comment]
I am not familiar with Scribble, but from what I see, it probably conflicts with the Orbinaut's renderer. I'd guess it resets the shader when calling text draw function. I am not providing support for third-party libraries, Orbinaut is designed to work with stock GameMaker

**Reply by ManeaterMildred** [reply, marked as answer]
That's fair, can't blame you for not wanting to handle third party libraries, haha. Thanks, I'll see if I can figure something out!
